widget/entry.go

```diff
@@ -304,3 +313,11 @@ func NewEntry() *Entry {
 	Renderer(e).Layout(e.MinSize())
 	return e
 }
+
+// NewEntryPassword creates a new entry password widget
+func NewEntryPassword() *Entry {
```

Comment:
Thinking from an API user perspective I feel that NewPasswordEntry would read better

Reply:
Updated in 7dbca9c285a4b71bbc03b5f88e81462d6c735072